dataactcore/models/stagingModels.py

```diff
@@ -53,6 +36,7 @@ class Appropriation(Base):
     tas = Column(Text, index=True, nullable=False, default=concatTas, onupdate=concatTas)
     tas_id = Column(Integer, ForeignKey("tas_lookup.tas_id", name='fk_tas'),
                     nullable=True)
+    tas_obj = relationship(TASLookup)
```

Comment:
Turns out, this also resolves a runtime dependency issue where sqlalchemy didn't know about the `tas_lookup` table when it processed the staging models. Flipping the imports and adding this relation ensured that they'd be processed in the right order at runtime, too.